scipy/stats/tests/test_stats.py

```diff
@@ -7486,3 +7490,25 @@ def test_input_validation(self):
         with assert_raises(TypeError, match="`ranked` must be boolean."):
             stats.page_trend_test(data=[[1, 2, 3], [1, 2, 3]],
                                   ranked="ekki")
+
+
+rng = np.random.default_rng(902340982)
+x = rng.random(10)
+y = rng.random(10)
+
+
+@pytest.mark.parametrize("fun, args",
+                         [(stats.wilcoxon, (x,)),
+                          (stats.ks_1samp, (x, stats.norm.cdf)),
```

Comment:
```suggestion
                          (stats.ks_1samp, (x, stats.norm.cdf)),  # type: ignore[<code>] # noqa
```
I'm assuming this is the source of:
`scipy/stats/tests/test_stats.py:7528: error: Module has no attribute "norm"  [attr-defined]`
Of course, line 7528 doesn't exist, but this is the only appearance of `stats.norm` in the file.